src/content/blog/docs-hacktoberfest.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: Hacktoberfest with Astro Docs
+description: 'It’s Astro Docs week! See how we hacked Hacktoberfest to celebrate our awesome contributors this October.'
+publishDate: 'October 31, 2022'
+socialImage: '/assets/blog/docs-hacktoberfest/cover.jpg'
+coverImage: '/assets/blog/docs-hacktoberfest/cover.jpg'
+lang: 'en'
+authors:
+    - chris
+    - sarah
+    - dan
+---
+
+import BlogContentImage from '/src/components/BlogContentImage.astro'
+import community from '/src/assets/blog/docs-hacktoberfest/contributors.jpg'
+
+Welcome to Astro Docs Week! All week, our docs maintainers are taking over [Astro’s Twitter account](https://twitter.com/astrodotbuild). We’ll be sharing docs updates and fun from the community, so make sure you’re following us!
+
+To kick things off, let’s take a moment to talk about Astro Docs and Hacktoberfest. We love to celebrate Astro’s amazing open-source contributors. Here’s how Hacktoberfest helped us do that.
+
+## Community-driven docs
+
+<BlogContentImage
+    src={community}
+    alt="Avatars for the many, many contributors to Astro docs"
+/>
+
+[Astro’s documentation website](https://docs.astro.build/) is open source and driven by our community. Our [270+ contributors](https://github.com/withastro/docs/graphs/contributors) make it easier for you to use Astro every day and we couldn’t do it without them! So this year, we wanted to use Hacktoberfest as a chance to recognize the work they do.
+
+[Hacktoberfest](https://hacktoberfest.com/) is an annual event run by Digital Ocean to encourage open-source participation. Every October, people who sign up can see their PRs rewarded with a t-shirt — or have a tree planted in their name! For the first time, this year’s edition welcomed [low or non-code contributions](https://hacktoberfest.com/about/#low-or-non-code). How could we say no to a forest of docs contributions?
+
+## Planting the seed
+
+For us, contributions come in all shapes and sizes. From typo fixes and CSS tweaks, to bigger documentation improvements. But that’s not all! Astro’s Docs are available in 10 languages thanks to our tireless translators. That’s where we decided to focus our efforts this month.
+
+In line with Hacktoberfest’s low-code focus, we wanted to encourage new translations. But **we can’t merge translations without reviews from our multilingual community.** It might not be obvious, but a good PR review is as valuable as the PR itself. To recognize this, we made it possible to [get Hacktoberfest credit for reviewing PRs](https://github.com/withastro/docs/blob/main/.github/hacktoberfest.md). Then we let folks know our plans on [Discord](https://astro.build/chat) and [Twitter](https://twitter.com/astrodotbuild/status/1576221838950412289), sat back, and waited…
+
+## So, how did it go?
+
+Drum roll please... Astro’s docs merged **more than 130 PRs from 57 unique contributors** as part of this year’s Hacktoberfest! This includes **40 first-time contributors** to the docs repo. 🎉
```

Comment:
```suggestion
Drum roll please... Astro’s docs merged **more than 130 PRs from 58 unique contributors** as part of this year’s Hacktoberfest! This includes **41 first-time contributors** to the docs repo. 🎉
```
Assuming we merge this StackBlitz one before this is posted.  😅

Reply:
Yay! 🎉 

Going to hold off committing until https://github.com/withastro/docs/pull/1960 actually gets merged just in case it doesn’t in time, but definitely want to if it does.

Reply:
Merged! 🥳